bool[] coord
float64[] sollPos
float64 T_traj
float64 acc_perc
bool TaskSpace
uint32 planAlgoSel
